web/src/components/TaskCtaArea.tsx

```diff
@@ -0,0 +1,123 @@
+import { PrimaryButtonDropdown } from "@/components/njwds-extended/PrimaryButtonDropdown";
+import { TaskCTA } from "@/components/TaskCTA";
+import { useUserData } from "@/lib/data-hooks/useUserData";
+import { PostOnboarding, Task } from "@/lib/types/types";
+import { openInNewTab } from "@/lib/utils/helpers";
+import { ProfileData } from "@businessnjgovnavigator/shared/index";
+import { useRouter } from "next/router";
+import React, { ReactElement } from "react";
+
+interface ButtonsArrayElement {
+  text: string;
+  destination: string;
+  onClick: () => void;
+}
+
+const TaskFooter = (props: React.ComponentPropsWithoutRef<"div">): JSX.Element => {
+  const { children, ...attributes } = props;
+  return (
+    <div
+      data-testid="cta-area"
+      style={{ marginTop: "2rem" }}
+      className="flex flex-justify-end bg-base-lightest margin-x-neg-4 padding-3 margin-bottom-neg-4 flex-column mobile-lg:flex-row radius-bottom-lg"
+      {...attributes}
+    >
+      <div className="mobile-lg:margin-bottom-0 margin-bottom-1">{children}</div>
+    </div>
+  );
+};
+
+interface Props {
+  postOnboardingQuestion: PostOnboarding | undefined;
+  task?: Task | undefined;
+  forceDisplay?: boolean | undefined;
+  onboardingKey?: keyof ProfileData | undefined;
+}
+
+export const TaskCtaArea = (props: Props): ReactElement => {
+  const ctaButtons: ButtonsArrayElement[] = [];
+  const { business } = useUserData();
+  const router = useRouter();
+
+  const taskCtaDestination = props.task?.callToActionLink;
+  let taskCtaText = props.task?.callToActionText ?? "";
+
+  if (taskCtaText && taskCtaText.length === 0) {
+    taskCtaText = "Start Application";
```

Comment:
[pebble] 

a. The `taskCtaText.length === 0` part of this is redundant, right?

b. What is the purpose of this statement? I don't see a test for it, and I'm surprised to see a bare string in the code - if this is desired behavior, it should be in Config. Unless this was left in by mistake?

Reply:
a. It is not; I'm *specifically* testing for the case where the length is 0 so that I can set some fallback text to display for the CTA button.

b. You're right about the bare string. I've updated it to pull this value from `Config`.

Reply:
Maybe at a higher level - why do we want fallback text for this? I didn't think that existed previously, and it's not part of the story?

Reply:
Not sure if this was part of the model change making this optional, but yes, this value existed previously. I pulled this from L33 of `TaskCTA.tsx`.

Reply:
But also wait... I'm still confused. I don't see how this could ever evaluate to true? 

`taskCtaText && taskCtaText.length === 0`

taskCtaText is either the text of the task, or empty string, according to the prev line. Empty string evaluates to falsey, so if it's empty, then `taskCtaText` would evaluate to false. And if it's not empty, then `taskCtaText.length === 0` would evaluate to false, so the && case can never be true. 

Am I missing something? If I am, I think a test case for this would clear up the confusion